esp/scm/ws_esdlconfig.ecm

```diff
@@ -25,20 +25,42 @@ ESPStruct MethodConfig
 {
     String Name;
     ESParray<ESPstruct NamedValue, Attribute> Attributes;
+    String Elements;
```

Comment:
MethodConfig was added in 1.2, and all uses of the structure are versioned, so I think we're fine.